README.md

```diff
@@ -252,7 +252,7 @@ As a result, you save gigabytes of space on your disk and you have a lot faster
 If you'd like more details about the unique `node_modules` structure that pnpm creates and
 why it works fine with the Node.js ecosystem, read this small article: [Flat node_modules is not the only way](https://pnpm.io/blog/2020/05/27/flat-node-modules-is-not-the-only-way).
 
-💖 Like this project? Let people know with a [tweet](https://r.pnpm.io/tweet)
+💖 Like this project? Let people know with a [X](https://x.com/pnpmjs) or [Bluesky](https://bsky.app/profile/pnpm.io)
```

Comment:
"with a X or Bluesky" makes no sense lmao.

Reply:
Do you mean we don't need to add the bluesky link?

Reply:
No. I mean the sentence. This would make some sense.

```suggestion
💖 Like this project? Follow pnpm on [X](https://x.com/pnpmjs) or [Bluesky](https://bsky.app/profile/pnpm.io)
```

Reply:
But both links have deviated from the original intention. The link `https://r.pnpm.io/tweet` would open X with a draft post prefilled with words to promote pnpm, waiting for the user to press "Post".

The new links just point the user to the pnpm profile on X and Bluesky. This purpose is already fulfilled (for X) by the links right under `Getting Started`. So you should add the Bluesky link to under `X` under `Getting Started`.

Reply:
That makes sense, I'll edit it later. By the way, the link above doesn't seem to need to be edited?

Reply:
> By the way, the link above doesn't seem to need to be edited?

Yep.